pkg/pillar/types/global.go

```diff
@@ -308,6 +308,14 @@ const (
 	// FmlCustomResolution global setting key
 	FmlCustomResolution GlobalSettingKey = "app.fml.resolution"
 
+	// Log filtering and dedupliction
+	// LogDedupWindowSize is a measure of how many log entries are saved to search for duplicates
+	LogDedupWindowSize GlobalSettingKey = "log.dedup.window.size"
+	// LogFilenamesToCount a comma-separated list of log filenames to count
+	LogFilenamesToCount GlobalSettingKey = "log.count.filenames"
```

Comment:
From the description, it's still not that clear how exactly this deduplication works... Could we add some documentation around the feature (maybe here: https://github.com/lf-edge/eve/blob/master/docs/LOGGING.md)? 
Also, the new options are to be described here: https://github.com/lf-edge/eve/blob/master/docs/CONFIG-PROPERTIES.md

Reply:
you're absolutely right! I added the docs now.